build/yamls/base/controller-rbac.yml

```diff
@@ -216,6 +216,14 @@ rules:
       - get
       - watch
       - list
+  - apiGroups:
+      - crd.antrea.io
+    resources:
+      - ippools
```

Comment:
seems ippools are using the same verbs as `externalippools` in the same group, you can just add it in the same place as below, no need to add extra section.
```
  - apiGroups:
      - crd.antrea.io
    resources:
      - externalippools
      - ippools
    verbs:
      - get
      - watch
      - list
```

Reply:
Thanks, done